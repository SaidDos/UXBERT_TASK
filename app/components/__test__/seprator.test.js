import 'react-native';
import React from 'react';
import Seprator from '../views/seprator';

import renderer from 'react-test-renderer';

it('should renders correctly', () => {
  const tree = renderer.create(<Seprator />).toJSON();
  expect(tree).toMatchSnapshot();
});
