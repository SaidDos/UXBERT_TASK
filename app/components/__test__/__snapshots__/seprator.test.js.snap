// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "lightgrey",
        "height": 1,
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
